packages/compass-data-modeling/src/components/diagram-editor.tsx

```diff
@@ -183,6 +188,35 @@ const DiagramContent: React.FunctionComponent<{
     });
   }, []);
 
+  // Center on a new collection when it is added
+  const previouslyOpenedDrawer = useRef<boolean>(false);
+  useEffect(() => {
```

Comment:
I know I will never convince y'all to avoid useEffect usage as much as possible 😆 but I just have to mention that while yes, this is an effect of sorts, I think a better way to think about this as (at least as it is right now) a side-effect directly of calling the createNewCollection action and so can be just a function that is called right after the `onAddCollectionClick` callback prop